sdk/communication/azure-communication-email/README.md

```diff
@@ -49,20 +49,16 @@ and then include the direct dependency in the dependencies section without the v
 #### Include direct dependency
 
 If you want to take dependency on a particular version of the library that is not present in the BOM,
-add the direct dependency to your project as follows.
-
-[//]: # ({x-version-update-start;com.azure:azure-communication-email;current})
+add the direct dependency to your project as follows. "VERSION" below should be the version number you want. Example: 1.0.22
 
 ```xml
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-communication-email</artifactId>
-    <version>1.2.0-beta.1</version>
+    <version>[VERSION]</version>
 </dependency>
 ```
 
-[//]: # ({x-version-update-end})
-
 ## Key concepts
 
 > More details coming soon.
```

Comment:
Put these under the `Key Concepts` section:


The email SDK provides two clients, a synchronous client and an asynchronous client:

EmailClient and EmailAsyncClient provide an operation to send an email message (plain text or html). Inline images are supported for HTML formatted email message, making emails appealing for marketing campaigns.
EmailClient and EmailAsyncClient also provide an operation to fetch the status of an email message sent.